App.Goods/Services/ValidateService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using App.Configuration;
+using App.Goods.Interfaces;
+
+namespace App.Goods.Services
+{
+    public class ValidateService : IValidateService, ITransientDependency
+    {
+        private readonly IProductsManager _productsManager;
+
+        public ValidateService(IProductsManager productsManager)
+        {
+            _productsManager = productsManager;
+        }
+
+        public IEnumerable<int> ValidateIds(int[] ids)
```

Comment:
The name "ValidateIds" is not descriptive enough. The actual purpose of this class, according to implementation is to "Validate product ids for order creation". Also this class do nothing if receive invalid Ids. 
I think it`s better for this class to:
1) return nothing. Because commonly Validator classes return nothing or return object, which describes validation errors
2) throw exception if unsupported id was received